graphdatascience/query_runner/neo4j_query_runner.py

```diff
@@ -205,18 +213,18 @@ def run_retryable_cypher(
         query: str,
         params: Optional[dict[str, Any]] = None,
         database: Optional[str] = None,
-        custom_error: bool = True,
         mode: Optional[QueryMode] = None,
+        custom_error: bool = True,
```

Comment:
this positional change, is that technically breaking? I'm not too concerned for this method but I think it would break code with positional parameters